city_scrapers/spiders/il_arts_council.py

```diff
@@ -0,0 +1,130 @@
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        for table in response.xpath('//table/tbody'):
+            year = self._get_year(table)
+            for item in table.xpath('./tr')[1::]:
+                meeting = Meeting(
+                    title=self._parse_title(item),
+                    description=self._parse_description(item),
+                    classification=self._parse_classification(item),
+                    start=self._parse_start(item, year),
+                    end=self._parse_end(item),
+                    all_day=self._parse_all_day(item),
+                    time_notes=self._parse_time_notes(item),
+                    location=self._parse_location(item),
+                    links=self._parse_links(item),
+                    source=self._parse_source(response),
+                )
+
+                meeting["status"] = self._get_status(item,
+                                                     text=item.xpath('td[2]/text()').get())
+                meeting["id"] = self._get_id(meeting)
+
+                yield meeting
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title."""
+        return "Board Meeting"
+
+    def _parse_description(self, item):
+        """Parse or generate meeting description."""
+        return "llinois Arts Council Agency Board Meeting"
+
+    def _parse_classification(self, item):
+        """Parse or generate classification from allowed options."""
+        return BOARD
+
+    def _parse_start(self, item, year):
+        """Parse start datetime as a naive datetime object."""
+        item_text = item.xpath('./td/text()').get()
+        if not item_text:
+            item_text = item.xpath('./td/a/text()').get()
+        text_clean = unicodedata.normalize("NFKD", item_text).strip()
+
+        if text_clean[-4::] == year:
+            start = dt.strptime(text_clean, '%A, %B %d, %Y')
+        elif text_clean[-2::] in ['am', 'pm']:
+            start = dt.strptime(text_clean + year, '%A, %B %d, %I:%M%p%Y')
+        else:
+            start = dt.strptime(text_clean + year, '%A, %B %d%Y')
+
+        return start
+
+    def _parse_end(self, item):
+        """Parse end datetime as a naive datetime object. Added by pipeline if None"""
+        return None
+
+    def _parse_time_notes(self, item):
+        """Parse any additional notes on the timing of the meeting"""
+        return ""
+
+    def _parse_all_day(self, item):
+        """Parse or generate all-day status. Defaults to False."""
+        return False
+
+    def _parse_location(self, item):
+        """Parse or generate location."""
+
+        location = item.xpath('td[2]/text()').get()
+        if not location:
+            location = item.xpath('td[2]/a/text()').get()
+
+        jrtc = {
+            "address":
+                "James R. Thompson Center, 100 West Randolph, Suite 10-500, Chicago, IL 60601",  # noqa
```

Comment:
We can take out the "James R. Thompson Center" part here and just start with "100 W"